comps/comps-katello-server-rhel7.xml

```diff
@@ -53,9 +53,6 @@
        <packagereq type="default">tfm-rubygem-qpid_messaging</packagereq>
        <packagereq type="default">tfm-rubygem-hammer_cli_csv</packagereq>
 
-       <!-- katello thirdparty non-rubygem packages -->
-       <packagereq type="default">python-urllib3</packagereq>
```

Comment:
You'll need to update this in foreman-packaging since I have moved it there (https://github.com/Katello/katello-packaging/pull/569)